nifi/1.10/0.1.0-1.5.0/install/index.md

```diff
@@ -0,0 +1,300 @@
+---
+post_title: Install and Customize
+menu_order: 20
+enterprise: 'no'
+---
+
+ DCOS Nifi is available in the Universe and can be installed by using either the web interface or the DC/OS CLI.
+
+The default DC/OS NiFi Service installation provides reasonable defaults for trying out the service, but that may not be sufficient for production use. You may require different configurations depending on the context of the deployment.
+
+
+## Configuration Best Practices for Production
+  
+    - Increase the number of TCP socket ports available
+    
+      This is particularly important if the flow will be setting up and tearing down a large number of sockets in small period of time.
+        
+           sudo sysctl -w net.ipv4.ip_local_port_range="10000 65000"
+       
+    - Tell Linux you never want NiFi to swap
+       Swapping is fantastic for some applications. It isn’t good for something like NiFi that always wants to be running. 
+       To tell Linux you’d like swapping off you can edit '/etc/sysctl.conf' to add the following line
+           
+           vm.swappiness = 0
+           
+        For the partitions handling the various NiFi repos turn off things like 'atime'. Doing so can cause a surprising bump in 
+        throughput. Edit the '/etc/fstab' file and for the partition(s) of interest add the 'noatime' option.
+
+## Prerequisites
+   
+- If you are using Enterprise DC/OS, you may [need to provision a service account](https://docs.mesosphere.com/latest/security/service-auth/custom-service-auth/) before installing DC/OS NiFi Service. Only someone with `superuser` permission can create the service account.
```

Comment:
[need to provision a service account](https://docs.mesosphere.com/latest/security/service-auth/custom-service-auth/)
returns a 404.

Reply:
Change the link [need to provision a service account] to redirect to correct URL. Also we created a page serviceaccountdetail.md and added it to "Tip: A complete guide to Configuring DC/OS Access for Nifi can be found here." On clicking [here], it will redirect to the new page for for configuring access in NiFi.